File: utils/base62.go
package utils

import "math/rand"

var base62Characters = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"

func GenerateBase62String(length int) string {
	var output string
	for len(output) < length {
		output += string([]rune(base62Characters)[rand.Intn(62)])
	}
	return output
}

File: ui/src/server/utils/base62.ts
const base62Characters = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";

export function generateBase62String(length: number): string {
	let output = "";
	while (output.length < length) {
		output += base62Characters.charAt(Math.random() * 61);
	}
	return output;
}
